Zend/zend_operators.c

```diff
@@ -1287,6 +1287,21 @@ ZEND_API zend_result ZEND_FASTCALL mul_function(zval *result, zval *op1, zval *o
 }
 /* }}} */
 
+static void ZEND_COLD zend_power_base_0_exponent_lt_eq_0_error(void)
+{
+	zend_error(E_DEPRECATED, "Power of base 0 and negative exponent is deprecated");
+}
+
+static zend_result safe_pow(double *result, double base, double exponent)
+{
+	if (UNEXPECTED(base == 0.0 && exponent < 0.0)) {
+		zend_power_base_0_exponent_lt_eq_0_error();
```

Comment:
Function name is inconsistent with the condition (< vs lt_eq)

Reply:
Thanks. Renamed function.